README.md

```diff
@@ -94,29 +100,38 @@ either the web or browser extension version of Ruffle.
 
 This project is tested with BrowserStack.
 
+### Android
+
+Follow the [instructions](https://github.com/ruffle-rs/ruffle-android/blob/main/CONTRIBUTING.md#building-from-source) in the `ruffle-android` project for building the Android application of Ruffle.
+
 ### Scanner
 
 If you have a collection of "real world" SWFs to test against, the scanner may be used to benchmark
 ruffle's parsing capabilities. Provided with a folder and an output filename, it will attempt to read
-all of the flash files and report on the success of such a task.
+all of the Flash files and report on the success of such a task.
 
 `cargo run --release --package=ruffle_scanner -- folder/with/swfs/ results.csv`
 
 ### Exporter
 
-If you have a swf and would like to capture an image of it, you may use the exporter tool.
+If you have an SWF file and would like to capture an image of it, you may use the exporter tool.
```

Comment:
I know this can be a source of contention, but I'm a firm believe in "**a** SWF" - as "SWF" has an official pronunciation of "swiff"